tests/commands/test_store_commands.py

```diff
@@ -1726,6 +1777,35 @@ def test_fetchlib_simple_downloaded(caplog, store_mock, tmp_path, monkeypatch, c
     assert saved_file.read_text() == lib_content
 
 
+def test_fetchlib_simple_dash_in_name(caplog, store_mock, tmp_path, monkeypatch, config):
```

Comment:
Please add a test for "fetchlib" using a path already in disk with underscore (NOT passing the `library` as a parameter, so it needs to pickup what's on disk to know what to fetch)